flink-core/src/main/java/org/apache/flink/util/function/FunctionWithThrowable.java

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
I think this commit should be before `[FLINK-10255] Only react to onAddedJobGraph signal when being leader`, or it should be squashed.

Reply:
Yes, I will rearrange the commits before merging. Locally 3a07ee8812afb3feb2233d722311c4de08eef09a is before 868c7dd6822cee24046b486874fef5d5d7d3564d which makes it work.